test/integration/protocol_integration_test.h

```diff
@@ -18,6 +18,27 @@ class ProtocolIntegrationTest : public HttpProtocolIntegrationTest {
              << "Please use DownstreamProtocolIntegrationTest instead of ProtocolIntegrationTest.";
     }
   }
+
+  void expectWireBytesSentAndReceived(std::string log_name, int log_index, int h1_wire_bytes_sent,
+                                      int h1_wire_bytes_received, int h2_wire_bytes_sent,
+                                      int h2_wire_bytes_received) {
+    auto integer_near = [](int x, int y) -> bool { return std::abs(x - y) <= (x / 20); };
```

Comment:
Please add comment that the exact count is not possible in H/2 due to variable compression of the `x-request-id` header.